test/dlc_tests/benchmark/ec2/pytorch/training/test_performance_pytorch_training.py

```diff
@@ -34,9 +34,13 @@
 PT_PERFORMANCE_TRAINING_GPU_IMAGENET_CMD = os.path.join(
     CONTAINER_TESTS_PREFIX, "benchmark", "run_pytorch_training_performance_gpu_imagenet"
 )
+PT_PERFORMANCE_TRAINING_GPU_INDUCTOR_CMD = os.path.join(
+    CONTAINER_TESTS_PREFIX, "benchmark", "run_pytorch_training_performance_gpu_inductor"
+)
 
 PT_EC2_GPU_SYNTHETIC_INSTANCE_TYPE = "p3.16xlarge"
 PT_EC2_GPU_IMAGENET_INSTANCE_TYPE = "p3.16xlarge"
+PT_EC2_GPU_INDUCTOR_INSTANCE_TYPES = ("p3.16xlarge", "p4d.24xlarge", "g5.48xlarge")
```

Comment:
At the moment, we are focused on single GPU tests. Can you downsize these instance types to g5.4xl and p3.2xl. 
Also please add g4dn.4xl as well.